### PR TITLE
CIT-31 #time 5h Create the bookForm app in Book_store project, submit  the data in the database via form and fetch the data from the database and display the data

### PR DESCRIPTION
**Pull request by codal-uchoudhary**
… the data in the database via form and fetch the data from the database and display the data

### CODE REVIEW

File: Projects/Book_store/bookStore/bookForm/models.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+# Create your models here.
+
+class Users(models.Model):
+    username = models.CharField(max_length = 40)
```
Comment (arvish-codal) [inline Projects/Book_store/bookStore/bookForm/models.py:6]: @codal-uchoudhary , shouldn't we include password as well in here ? apart from username.

I believe this model is used in the Sign-Up endpoint right ?

Comment (arvish-codal) [reply Projects/Book_store/bookStore/bookForm/models.py:6]: Check this --> https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html

in the above link, read from the following section

Extending User Model Using a Custom Model Extending AbstractUser

Comment (codal-uchoudhary) [reply Projects/Book_store/bookStore/bookForm/models.py:6]: Actually , this is not for authentication purpose. for practice , i just submit the input data in database via form.

Comment (codal-uchoudhary) [reply Projects/Book_store/bookStore/bookForm/models.py:6]: and again fetch data from database and display the data .

**Review by arvish-codal — APPROVED** [review]

**Pull request merged**